corpusreaders/src/main/java/edu/illinois/cs/cogcomp/nlp/corpusreaders/ontonotes/ConvertOntonotesToColumn.java

```diff
@@ -0,0 +1,160 @@
+    static final private Set<String> dropTags = new HashSet<>();
+
+    // define the attributes we want to keep for the tags we have.
+    static final private Map<String, Set<String>> tagsWithAtts = new HashMap<>();
+    static {
+        Set<String> docAttrs = new HashSet<>();
+        docAttrs.add("docno");
+        tagsWithAtts.put("doc", docAttrs);
+        Set<String> nameAttrs = new HashSet<>();
+        nameAttrs.add("type");
+        tagsWithAtts.put("enamex", nameAttrs);
+    }
+
+    /**
+     * read the file indicated by the argument which is the file name, and path.
+     * @param document the data read from the file.
+     * @return the XmlTextAnnotation containing the text annotation, and xml markup offset data.
+     * @throws IOException 
+     */
+    static private XmlTextAnnotation getNameTextAnnotation(File file) throws IOException {
+        String document = LineIO.slurp(file.getCanonicalPath());
+        // we keep everything.
+        XmlDocumentProcessor xmlProcessor = new XmlDocumentProcessor(tagsWithText, tagsWithAtts, dropTags, true);
+        StatefulTokenizer st = new StatefulTokenizer();
+        TokenizerTextAnnotationBuilder taBuilder = new TokenizerTextAnnotationBuilder(st);
+        XmlTextAnnotationMaker xtam = new XmlTextAnnotationMaker(taBuilder, xmlProcessor);
+ 
+        // read the file and create the annotation.
+        XmlTextAnnotation xta = xtam.createTextAnnotation(document, "OntoNotes 5.0", "test");
+        TextAnnotation ta = xta.getTextAnnotation();
+        List<SpanInfo> fudge = xta.getXmlMarkup();
+ 
+        // create the named entity vi
+        View nerView = new SpanLabelView(ViewNames.NER_ONTONOTES, ta);
+        for (SpanInfo si : fudge) {
+            if ("enamex".equalsIgnoreCase(si.label)) {
+ 
+                IntPair charOffsets = si.spanOffsets;
+                String neLabel = si.attributes.get("type").getFirst();
+                int cleanTextCharStart = xta.getXmlSt().computeModifiedOffsetFromOriginal(charOffsets.getFirst());
+                int cleanTextCharEnd = xta.getXmlSt().computeModifiedOffsetFromOriginal(charOffsets.getSecond());
+                int cleanTextNeTokStart = ta.getTokenIdFromCharacterOffset(cleanTextCharStart);
+                int cleanTextNeTokEnd = ta.getTokenIdFromCharacterOffset(cleanTextCharEnd-1); // StringTransformation returns one-past-the-end index; TextAnnotation maps at-the-end index
+                Constituent neCon = new Constituent(neLabel, nerView.getViewName(), ta, cleanTextNeTokStart, cleanTextNeTokEnd + 1); //constituent token indexing uses one-past-the-end
+                nerView.addConstituent(neCon);
+            }
+        }
+        ta.addView(ViewNames.NER_ONTONOTES, nerView);
+        return xta;
+    }
+    
+    /**
+	 * This is used primarily for quick testing. The bulk of this should be used as a
+	 * starting point for a unit test, question is can we include some of the files for
+	 * the test in the code repository?
```

Comment:
I would say: if you can take about 4 lines from each of 3 different docs, and have them be from different places within the docs (i.e. not simply the first 4 lines of each), this counts as fair use and should not offend LDC.